ap2016.css

```diff
@@ -224,7 +224,8 @@ h1, h2 {
 }
 
 .gold img{
-  height: 200px;
```

Comment:
removing the height makes the different logos look like they are a different size; perhaps keep a max-height to normalize them?